### PR TITLE
[6.16]Add test for All Hosts bulk errata apply

### PR DESCRIPTION
**Pull request by sambible**
Test exercising All Hosts Bulk Errata wizard, applying an erratum to multiple hosts. 

Requires: https://github.com/SatelliteQE/airgun/pull/1526

### CODE REVIEW

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_all_hosts_manage_errata'
airgun: 1526

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8238
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_all_hosts_manage_errata --external-logging
Test Result : ========= 2 passed, 72 deselected, 314 warnings in 1407.09s (0:23:27) ==========
```

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -2517,6 +2517,67 @@ def _cleanup():
...
+        result = host.run(f'rpm -q {FAKE_7_CUSTOM_PACKAGE}')
+        assert result.status == 0
+
+    with module_target_sat.ui_session() as session:
+        session.organization.select(module_org.name)
+        session.location.select(loc_name=DEFAULT_LOC)
+        session.all_hosts.manage_errata(
+            host_names=[mod_content_hosts[0].hostname, mod_content_hosts[1].hostname],
+            erratas_to_apply_by_id=[errata_id],
+            manage_by_customized_rex=False,
```
Comment (LadislavVasina1) [inline tests/foreman/ui/test_host.py:2567]: Shouldn't we also test the option when we use `manage_by_customized_rex=True` and the option when we apply more than 1 errata?

File: tests/foreman/ui/test_host.py [hunk 2]
```diff
@@ -2517,6 +2517,67 @@ def _cleanup():
...
+            'YumRepository': {'url': settings.repos.yum_3.url},
+        }
+    ],
+    ids=['yum3'],
+    indirect=True,
+)
+@pytest.mark.no_containers
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+@pytest.mark.rhel_ver_match('8')
```
Comment (vsedmik) [inline tests/foreman/ui/test_host.py:2534]:
```suggestion
@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
```

File: tests/foreman/ui/test_host.py (hunk 1 above)
Comment (vsedmik) [reply tests/foreman/ui/test_host.py:2567]: Maybe we could parametrize that.

File: tests/foreman/ui/test_host.py [hunk 3]
```diff
@@ -2517,6 +2517,67 @@ def _cleanup():
...
+):
+    """Apply an errata on multiple hosts through bulk errata wizard in All Hosts page.
+
+    :id: 41bca5f8-bf2d-4b16-974c-5abda3c09ee2
+
+    :expectedresults: Errata can be bulk applied to hosts through the All Hosts page.
+
+    :CaseComponent: Hosts-Content
+
+    :Team: Phoenix-content
```
Comment (vsedmik) [inline tests/foreman/ui/test_host.py:2551]:
```suggestion
    :Team: Phoenix-subscriptions
```
Hosts-Content belongs to the subs team.

**Review by vsedmik — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_host.py (hunk 1 above)
Comment (sambible) [reply tests/foreman/ui/test_host.py:2567]: I have a few more cases to add

File: tests/foreman/ui/test_host.py (hunk 3 above)
Comment (sambible) [reply tests/foreman/ui/test_host.py:2551]: This will never make sense to me......it's in the name.....

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_all_hosts_manage_errata'
airgun: 1526

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8257
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_all_hosts_manage_errata --external-logging
Test Result : ==== 2 failed, 2 passed, 71 deselected, 258 warnings in 1594.32s (0:26:34) =====
```

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_all_hosts_manage_errata'
airgun: 1526

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8259
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_all_hosts_manage_errata --external-logging
Test Result : ==== 2 failed, 2 passed, 71 deselected, 192 warnings in 1158.47s (0:19:18) =====
```

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_all_hosts_manage_errata'
airgun: 1526

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8260
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_all_hosts_manage_errata --external-logging
Test Result : ==== 1 failed, 3 passed, 71 deselected, 203 warnings in 1295.11s (0:21:35) =====
```

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_all_hosts_manage_errata'
airgun: 1526

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8261
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_all_hosts_manage_errata --external-logging
Test Result : ==== 2 failed, 2 passed, 71 deselected, 198 warnings in 1349.48s (0:22:29) =====
```

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_all_hosts_manage_errata'
airgun: 1526

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8262
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_all_hosts_manage_errata --external-logging
Test Result : ========= 4 passed, 71 deselected, 210 warnings in 1372.77s (0:22:52) ==========
```

**Review by damoore044 — APPROVED** [review]
looks good!

**Review by LadislavVasina1 — APPROVED** [review]
ACK

File: tests/foreman/ui/test_host.py [hunk 4]
```diff
@@ -2517,44 +2518,73 @@ def _cleanup():
...
-    ).create()
-    new_hostgroup = target_sat.api.HostGroup(
-        location=[module_location], organization=[module_org]
-    ).create()
-    for _ in range(3):
-        target_sat.api.Host(
-            organization=module_org, hostgroup=hostgroup, location=module_location
-        ).create()
-    with target_sat.ui_session() as session:
+    if errata_to_install == "1":
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:2559]:
```suggestion
    if errata_to_install == '1':
```

File: tests/foreman/ui/test_host.py [hunk 5]
```diff
@@ -2517,44 +2518,73 @@ def _cleanup():
...
+        errata_ids = settings.repos.yum_3.errata[25]
+    if errata_to_install == "2":
+        errata_ids = [settings.repos.yum_3.errata[25], settings.repos.yum_1.errata[1]]
+    for host in mod_content_hosts:
+        host.add_rex_key(module_target_sat)
+        module_repos_collection_with_setup.setup_virtual_machine(host, enable_custom_repos=True)
+        host.run(f'yum install -y {FAKE_7_CUSTOM_PACKAGE}')
+        result = host.run(f'rpm -q {FAKE_7_CUSTOM_PACKAGE}')
+        assert result.status == 0
+        if errata_to_install == "2":
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:2569]:
```suggestion
        if errata_to_install == '2':
```

File: tests/foreman/ui/test_host.py [hunk 6]
```diff
@@ -2517,44 +2518,73 @@ def _cleanup():
...
-                {
-                    'Host group': True,
-                }
+        session.location.select(loc_name=DEFAULT_LOC)
+        session.all_hosts.manage_errata(
+            host_names=[mod_content_hosts[0].hostname, mod_content_hosts[1].hostname],
+            erratas_to_apply_by_id=errata_ids,
+            manage_by_customized_rex=manage_by_custom_rex,
+        )
+        if errata_to_install == "2":
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:2581]:
```suggestion
        if errata_to_install == '2':
```

**Pull request merged**